src/tests/Interop/NativeLibrary/API/NativeLibraryTests.cs

```diff
@@ -179,6 +179,11 @@ public static int Main()
             success &= EXPECT(TryGetLibraryExport(handle, "NonNativeSum"), TestResult.ReturnFailure);
 
             NativeLibrary.Free(handle);
+
+            // -----------------------------------------------
+            //         GetLibraryExport Tests
+            // -----------------------------------------------
+            success &= EXPECT(GetEntryPointModuleHandle());
```

Comment:
I'd also like to see a negative test so we validate the exception code paths. Perhaps search for `"ExportThatDoesntExist"` too?